platform/core/src/studies/services/wado/retrieveMetadata.js

```diff
@@ -468,37 +468,70 @@ async function createStudyFromSOPInstanceList(server, sopInstanceList) {
  * @param {string} studyInstanceUid The Study Instance UID of the study which needs to be loaded
  * @returns {Object} A study descriptor object
  */
-async function RetrieveMetadata(server, studyInstanceUid) {
-  return (server.enableStudyLazyLoad !== false
+async function RetrieveMetadata(server, studyInstanceUid, filters) {
+  return (server.enableStudyLazyLoad !== false && false
     ? lazyLoadStudyMetadata
-    : loadStudyMetadata)(server, studyInstanceUid);
+    : loadStudyMetadata)(server, studyInstanceUid, filters);
 }
 
 /**
  *
  * @param {*} server
  * @param {*} studyInstanceUID
  */
-async function loadStudyMetadata(server, studyInstanceUID) {
-  const dicomWeb = new api.DICOMwebClient({
+async function loadStudyMetadata(server, studyInstanceUID, filters) {
```

Comment:
Update JSDOC

If it feels like I'm asking "what can be in `filters`"? too much, then that might be indicative of two things:

1. Why are we passing filters so far down the callstack? Does every layer of this beast have a good purpose? Does each actually need to know our filters?
2. Is there a better pattern for passing complex values?

Reply:
Definitely we can revisit and discuss about these many layers.. I dont see a purpose of having that many.
"filters" must be passed to the very end layer which does the call to retrieve series (in case of filter), because lazy load uses the first one (in our case would be the filtered) to trigger the process.

There is another logic which  we can place the filtering/promoting at the very beginning layer, but from my understanding this wouldnt be a good user experience at all (page will full load, than filter will be applied). I.e on a lazy env.. studies will be listed then filtered.

Let me know what you think?